Sources/_Datadog_Private/ObjcAppLaunchHandler.m

```diff
@@ -4,60 +4,94 @@
 * Copyright 2019-2020 Datadog, Inc.
 */
 
-#import <UIKit/UIKit.h>
 #import "ObjcAppLaunchHandler.h"
+#import <sys/sysctl.h>
+#import <UIKit/UIKit.h>
+
+// `AppLaunchHandler` aims to track some times as part of the sequence described in Apple's "About the App Launch Sequence"
+// https://developer.apple.com/documentation/uikit/app_and_environment/responding_to_the_launch_of_your_app/about_the_app_launch_sequence
 
-@interface AppLaunchTimer : NSObject
-@property NSDate *frameworkLoadTime;
-// Knowing how this value is collected and that it is written only once, the use of
-// `atomic` property is enough for thread safety. It guarantees that neither partial
-// nor garbage value will be returned.
-@property (atomic) NSTimeInterval timeToApplicationBecomeActive;
+// Note that the following properties will be set early enough in the App lifecycle to avoid any race conditions without any specific synchronization means.
+static NSTimeInterval TimeToApplicationFinishedLaunching = 0.0;
+static NSTimeInterval TimeToLoadApplicationRootUI = 0.0;
```

Comment:
I don't see any usages of this value - we only set it, but never read / process. Do we need it?

Reply:
You're right, I've sent this along for reference in case we consider this additional step eventually, but I'm going to clean this up in a new commit.